lua/fittencode/engines/actions.lua

```diff
@@ -239,12 +241,17 @@ function ActionsEngine.start_action(action, opts)
 
   Log.debug('sln: {}, eln: {}', sln, eln)
 
-  local vmode = { 'v', 'V', '<C-V>' }
   Log.debug('mode: {}', api.nvim_get_mode().mode)
-  if vim.tbl_contains(vmode, api.nvim_get_mode().mode) then
-    sln = fn.getpos("'<")[2]
-    eln = fn.getpos("'>")[2]
+  if vmode[api.nvim_get_mode().mode] then
```

Comment:
I use hash map to check if the vmode is matched, and I moved the definition out of the function.